src/app/src/App.js

```diff
@@ -43,6 +43,7 @@ import {
 library.add(
     faInfoCircle,
     fasInfoCircle,
+    faExclamationCircle,
```

Comment:
Out of curiosity, what does this change do?

Reply:
We have to individually import each font awesome icon we want to use ~ this one pulls in the (!) icon